packages/runner-ct/src/specs/specs-list.tsx

```diff
@@ -1,43 +1,49 @@
+import * as React from 'react'
 import { observer } from 'mobx-react'
-import React, { Component } from 'react'
 import State from '../lib/state'
 import { makeSpecHierarchy } from './make-spec-hierarchy'
 import { SpecItem } from './spec-item'
 
 interface SpecsListProps {
   state: State
+  config: Cypress.ConfigOptions
 }
 
-@observer
-class SpecsList extends Component<SpecsListProps> {
-  render () {
-    const { state } = this.props
-
+export const SpecsList: React.FC<SpecsListProps> = observer(
+  function SpecsList ({ state, config }) {
     const specGroups = state.filteredSpecs.length ? makeSpecHierarchy(state.filteredSpecs) : []
 
     return (
       <div className="specs-list">
         <header>
+          <h1>
+            { // @ts-expect-error config.project name is not typed
+              config.projectName ?? document.title ?? 'Cypress'
+            }
+          </h1>
+
+        </header>
+        <div className="specs-list-search-input-container">
           <input
             placeholder='Select tests to run...'
             value={state.specSearchText}
-            onChange={(e) => this.props.state.setSearchSpecText(e.currentTarget.value.toLowerCase())}
+            onChange={(e) => state.setSearchSpecText(e.currentTarget.value.toLowerCase())}
           />
-        </header>
-        <ul className="specs-list-container">{
-          specGroups.map((item) => {
-            { // The `active` prop here is used only to
-              // force repaint of the tree when selecting a spec
-              // It is not used for anything else than to patch react
-              // not comparing members of an object (this.props.state in this case)
-            }
+        </div>
+        <div className="specs-list-scroll-container">
+          <ul className="specs-list-container">{
+            specGroups.map((item) => {
+              { // The `active` prop here is used only to
```

Comment:
Is this comment still relevant? I cannot see an `active` prop.

Also this seems weird, is this a common problem in mobx (trick react into repainting?)

Reply:
No idea, I just refactored to ts